File: Punto2/Estudiante.java
public class Estudiante {
    String nombre;
    int edad;
    double promedio;

    public Estudiante(String nombre, int edad, double promedio) {
        this.nombre = nombre;
        this.edad = edad;
        this.promedio = promedio;
    }

    public static void main(String[] args) {
        Estudiante[] estudiantes = {
            new Estudiante("Carlos Gomez", 20, 4.5),
            new Estudiante("Maria Rodriguez", 22, 4.8),
            new Estudiante("Juan Perez", 19, 3.9)
        };

        for (Estudiante e : estudiantes) {
            System.out.println("Nombre: " + e.nombre + ", Edad: " + e.edad + ", Promedio: " + e.promedio);
        }
    }
}

File: Punto2/estudiante.go
package main

import "fmt"

type Estudiante struct {
	Nombre   string
	Edad     int
	Promedio float64
}

func main() {
	estudiantes := []Estudiante{
		{Nombre: "Carlos Gomez", Edad: 20, Promedio: 4.5},
		{Nombre: "Maria Rodriguez", Edad: 22, Promedio: 4.8},
		{Nombre: "Juan Perez", Edad: 19, Promedio: 3.9},
	}

	for _, e := range estudiantes {
		fmt.Printf("Nombre: %s, Edad: %d, Promedio: %.1f\n", e.Nombre, e.Edad, e.Promedio)
	}
}
